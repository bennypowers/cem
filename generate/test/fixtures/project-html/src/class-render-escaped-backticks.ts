@customElement('class-render-escaped-backticks')
class ClassRenderEscapedBackticks extends LitElement {
  render() {
    return html`
      <!-- \`<slot>\` -->
      <slot></slot>
      <!-- summary: "\`<slot>\`" -->
      <slot name="slot"></slot>
      <!-- \`<p>\` -->
      <p part="part"></p>
      <!-- summary: "\`<p>\`" -->
      <p part="summary"></p>
    `;
  }
}
