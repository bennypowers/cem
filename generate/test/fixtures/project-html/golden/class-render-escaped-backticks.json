{
  "schemaVersion": "2.1.0",
  "modules": [
    {
      "kind": "javascript-module",
      "path": "src/class-render-escaped-backticks.js",
      "declarations": [
        {
          "name": "ClassRenderEscapedBackticks",
          "superclass": {
            "name": "LitElement",
            "package": "lit"
          },
          "kind": "class",
          "tagName": "class-render-escaped-backticks",
          "slots": [
            {
              "name": "",
              "description": "`\u003cslot\u003e`"
            },
            {
              "name": "slot",
              "summary": "`\u003cslot\u003e`"
            }
          ],
          "cssParts": [
            {
              "name": "part",
              "description": "`\u003cp\u003e`"
            },
            {
              "name": "summary",
              "summary": "`\u003cp\u003e`"
            }
          ],
          "customElement": true
        }
      ],
      "exports": [
        {
          "kind": "custom-element-definition",
          "name": "class-render-escaped-backticks",
          "declaration": {
            "name": "ClassRenderEscapedBackticks",
            "module": "src/class-render-escaped-backticks.js"
          }
        }
      ]
    }
  ]
}